src/main/resources/assets/immersivepetroleum/lang/zh_cn.json

```diff
@@ -11,41 +11,61 @@
 	
 	"block.immersivepetroleum.asphalt": "沥青混凝土",
 	"block.immersivepetroleum.asphalt_slab": "沥青混凝土台阶",
-	"block.immersivepetroleum.asphalt_stairs": "沥青混凝土楼梯",
+	"block.immersivepetroleum.asphalt_stair^": "沥青混凝土楼梯",
```

Comment:
Just remove `^` then, it isnt in the original either. Don't know why you guys put it in there :D